modules/standard/Time.chpl

```diff
@@ -2182,22 +2196,17 @@ proc getCurrentTime(unit: TimeUnits = TimeUnits.seconds) : real(64) do
    The month is in the range 1 to 12.
    The day   is in the range 1 to 31
 */
-@deprecated(notes="'getCurrentDate' is deprecated; access the individual fields of 'date.today()' as needed instead")
+@deprecated(notes="'getCurrentDate' is deprecated; access the individual fields of 'date.utcToday()' as needed instead, or use 'date.today()' for local wall time")
 proc getCurrentDate() : (int, int, int) {
-  var now = chpl_now_timevalue();
-
-  var seconds, minutes, hours, mday, month, year, wday, yday, isdst:int(32);
-
-  chpl_timevalue_parts(now, seconds, minutes, hours, mday, month, year, wday, yday, isdst);
-
-  return (year + 1900, month + 1, mday);
+  var today = date.utcToday();
+  return (today.year, today.month, today.day);
 }
 
 /*
-   :returns: The current day of the week
+   :returns: The current day of the week, calculated from UTC time.
    :rtype:   :type:`day`
  */
-@deprecated("`getCurrentDayOfWeek` is deprecated; please use `date.today().weekday()` instead")
+@deprecated("`getCurrentDayOfWeek` is deprecated; please use `date.utcToday().weekday()` instead, or `date.today().weekday()` for local wall time")
```

Comment:
nitpick: I'm not sure which style I prefer, but can you either use `` ` `` or `'` so the deprecation are consistent?

Reply:
Sure, I think I'd go with `'` since in my mind backticks are for when we can make `:proc:` links which we couldn't here.